src/ts/mapboxMap.ts

```diff
@@ -200,13 +200,15 @@ module powerbi.extensibility.visual {
 
             //check for GL Support
             // if (!mapboxgl.supported()) {
-            //     this.errorDiv.innerHTML = Templates.noGlSupport
+            //     const text = document.createTextNode(Templates.noGlSupport);
+            //     this.errorDiv..appendChild(text);
```

Comment:
Is there an intention to show this error message?

If we aren't checking for support and displaying anything, I would recommend just removing the code. It can be added back later (using git history to assist if needed).